lib/watir_drops/page_object.rb

```diff
@@ -18,52 +20,68 @@ def page_url
         end
       end
 
+      def page_title
+        define_method("page_title") do |args|
+          yield(*args)
+        end
+      end
+
       def element_list
         @element_list ||= []
       end
 
+      def required_element_list
+        @required_element_list ||= []
+      end
+
       def inherited(subclass)
         subclass.element_list = element_list.dup
+        subclass.required_element_list = required_element_list.dup
       end
 
-      def elements(name, &block)
+      def elements(name, required = false, &block)
         define_method(name) do |*args|
           self.instance_exec(*args, &block)
         end
 
         element_list << name.to_sym
+        required_element_list << name.to_sym if required
       end
 
-      def element(name, &block)
+      def element(name, required=false, &block)
```

Comment:
Also, contemplating the naming. Which is more descriptive `unique` or `required`?